### PR TITLE
LANraragi 0.6.9 (new formula)

### PR DESCRIPTION
**Pull request by Bl4Cc4t**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Notes:
A brewed `libarchive` is needed, since the `archive.h` header is required for the program to work.

### CODE REVIEW

File: Formula/lanraragi.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require "language/node"
+
+class Lanraragi < Formula
+  desc "Web application for archival and reading of manga/doujinshi. Lightweight and Docker-ready for NAS/servers."
+  homepage "https://github.com/Difegue/LANraragi"
+  url "https://github.com/Difegue/LANraragi/archive/v.0.6.9-homebrew.tar.gz"
```
Comment (SMillerDev) [inline Formula/lanraragi.rb]: What's different about the homebrew version?

File: Formula/lanraragi.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}"
+      system "make"
+      system "make", "install"
+    end
+
+    system "npm", "install", *Language::Node.local_npm_install_args
+    system "perl", "./tools/install.pl", "install-full"
+
+    prefix.install "README.md"
```
Comment (SMillerDev) [inline Formula/lanraragi.rb]: Is there not a makefile that contains all this?

File: Formula/lanraragi.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    libexec.install "tests"
+    libexec.install "tools/build/homebrew/redis.conf"
+    libexec.install "lrr.conf"
+  end
+
+  test do
+    ENV["PERL5LIB"] = libexec/"lib/perl5"
+    ENV["LRR_LOG_DIRECTORY"] = testpath/"log"
+
+    system "npm", "--prefix", prefix, "test"
```
Comment (SMillerDev) [inline Formula/lanraragi.rb]: This doesn't seem to test any program functions.

File: Formula/lanraragi.rb (hunk 1 above)
Comment (Bl4Cc4t) [reply Formula/lanraragi.rb]: Only a directory change, see https://github.com/Difegue/LANraragi/compare/v.0.6.9...v.0.6.9-homebrew

File: Formula/lanraragi.rb (hunk 2 above)
Comment (Bl4Cc4t) [reply Formula/lanraragi.rb]: No, is that a problem?

File: Formula/lanraragi.rb (hunk 3 above)
Comment (Bl4Cc4t) [reply Formula/lanraragi.rb]: It does, this executes all the test files present in `#{prefix}/tests` using https://github.com/bigeasy/proof

**Comment by Bl4Cc4t** [comment]
@SMillerDev bump

**Comment by SMillerDev** [comment]
It needs to successfully build first.

**Comment by Bl4Cc4t** [comment]
Oh. Seems like I overlooked something. I’m sorry about that. I guess we have to wait yet another subrelease…

**Pull request closed without merging**